osu_standard.install

```diff
@@ -39,6 +39,64 @@ function osu_standard_install($is_syncing) {
 
 }
 
+/**
+ * Setup sticky toc
+ */
+function osu_standard_update_9013(&$sandbox) {
+  \Drupal::service('module_installer')->install([
+    'toc_js',
+    'toc_js_per_node',
+  ], TRUE);
+
+  $architect = Role::load('architect');
+  $architect->grantPermission('administer toc_js');
+  $architect->grantPermission('administer toc_js per node');
```

Comment:
lets sneak in a grant permission to 'administer easy breadcrumb' to the architect user as well.

Reply:
and I guess also add it to  the role "manage site configuration" as well